pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusCommunityDepAnalyzer.java

```diff
@@ -148,13 +148,20 @@ public void trigger() {
     }
 
     private Path buildReactiveProject(String settingsSelector) {
-        Path projectPath = generateQuarkusProject(artifactId -> artifactId.contains("reactive"), settingsSelector);
+        Path projectPath = generateQuarkusProject(
+                artifactId -> artifactId.contains("micrometer"),
+                artifactId -> !artifactId.contains("reactive,metrics"),
```

Comment:
should it be a check that  artifact id contains either `reactive` or `metrics`, not `reactive,metrics`?

Reply:
Yes, probably we can remove reactive and keep only micrometer in one condition and metrics in another, wdyt?